### PR TITLE
MANGO-1544: Integrate common UDMI library into UDMI project

### PR DESCRIPTION
**Pull request by erik-radixiot**
## Description
[MANGO-1544: Integrate common UDMI library into UDMI project](https://radixiot.atlassian.net/browse/MANGO-1544)

* Integrate library into UDMI fork and make the build go green checked on the tests.

## Changes:
- Fix some overrides to keep the same functionality as before

### CODE REVIEW

**Review by MertCingoz — COMMENTED** [review]
@erik-radixiot we also need to add this GitHub Action variable to run all the available tests.

`TARGET_PROJECT=//mqtt/localhost`

**Comment by erik-radixiot** [comment]
> @erik-radixiot we also need to add this GitHub Action variable to run all the available tests.
> 
> `TARGET_PROJECT=//mqtt/localhost`

@MertCingoz I don't have permission to set that variable (or set anything else) in Actions, and I believe it's not ok to hardcode it too... who can help me with that?

EDIT: it is [looking good so far](https://github.com/RadixIoT/udmi/actions/runs/10909581710/job/30278106558?pr=5) with the [env var hardcoded](https://github.com/RadixIoT/udmi/pull/5/commits/e2b71c6577f6554355a6f217835912ac89d05428), I'll remove the hardcoded env variable after approval, thanks.

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ on:
   schedule:
     - cron: '0 13 * * *'
 
+env:
```
Comment (erik-radixiot) [inline .github/workflows/testing.yml]: Remove this before merge

Comment (MertCingoz) [reply .github/workflows/testing.yml]: This was no effect. I had to hardcode variable to run tests and it is all green thanks!

**Review by MertCingoz — APPROVED** [review]

File: .github/workflows/testing.yml (hunk 1 above)
Comment (erik-radixiot) [reply .github/workflows/testing.yml]: Awesome, I'll revert those changes, thanks Mert.

**Pull request merged**